stetho/src/main/java/com/facebook/stetho/inspector/elements/android/DialogFragmentDescriptor.java

```diff
@@ -24,6 +24,7 @@
 import com.facebook.stetho.inspector.elements.DescriptorMap;
 import com.facebook.stetho.inspector.elements.NodeType;
 import com.facebook.stetho.inspector.elements.StyleAccumulator;
+import com.facebook.stetho.inspector.elements.ComputedStyleAccumulator;
```

Comment:
nit: to keep things nice and alphabetical, this should move up a few lines

(and there's a few more of these in the diff, I'll just leave this 1 comment for them)